CotEditor/en-GB.lproj/Credits.css

```diff
@@ -0,0 +1,7 @@
+
+*     { margin: 0 }
+body  { font: 11px/1.6 -apple-system, sans-serif; text-align: center;
+        color: -apple-system-label; background: -apple-system-background }
+h2    { font-size: 11px; font-weight: normal;
+        color: -apple-system-secondary-label }
+.hr   { margin: 1em }
```

Comment:
The same as Acknowledgments.css. Because the same name file in the Base.lproj directory is automatically used if not exist in the individual .lporj directory and the file does not contain any language-depended expression, this file can be removed.

Reply:
Fixed in f40005f